kythe/typescript/testdata/object.ts

```diff
@@ -1,14 +1,25 @@
 export {}
 
+const shortProperty = 0;
+
+//- @#0"computed" defines/binding Computed
+const computed = 'computed';
+
 const Object = {
   //- @property defines/binding Property
   //- Property.node/kind variable
   property: 3,
 
+  //- @shortProperty defines/binding ShortProperty
+  //- ShortProperty.node/kind variable
+  shortProperty,
+
+  //- @computed ref Computed
+  [computed]: 0,
+
   //- @method defines/binding Method
   //- Method.node/kind function
-  method() {
-  }
+  method() {}
 };
 
 //- @property ref Property
```

Comment:
Maybe some test of accessing shortProperty here too?

Reply:
Done. Also added xrefs to computed properties.